### ISSUE TITLE
Packages page

### ISSUE THREAD

**Issue by bil-elmoussaoui** [gnumdk/lollypop-web#7]
The packages page is not looking good at all.. i think just using a list for packages would be better than opening a new page..
also we should add elementary freya too

**Comment by wi1d5ky** [comment]
I think that elementary freya is one version of Ubuntu, so we don't need to add it again.
And I agree with you, using list is good.
Maybe we could use some js to display a list below the buttons when package button is clicked.

**Comment by bil-elmoussaoui** [comment]
well elementary is based on ubuntu, but elementary uses gtk 3.14 even if it's based on ubuntu 14.04 (that uses an older version)

**Comment by wi1d5ky** [comment]
ok, if you think it is needed, we could add it.
It is good for Freya users! ;)

**Comment by bil-elmoussaoui** [comment]
Do you know any nice js effect that we can use ? if not i think that using something like zoombox would be good too ?

**Comment by wi1d5ky** [comment]
I think pure JS is good

**Comment by wi1d5ky** [comment]
Do you want to take this issue? If you do, you can use what you want. ;)
I want to take the multi-language issue(#5). What do you think?

**Comment by bil-elmoussaoui** [comment]
If you want to take muli-language issue i will take this one

**Comment by wi1d5ky** [comment]
Or maybe you can use CSS-Trick!
Use `label`, `input[type="checkbox"]`, and css selector `+` to finish that staff

**Comment by bil-elmoussaoui** [comment]
I will see what i can do ;) i might push a fix tonight (it's 18h in my country :p )

**Comment by wi1d5ky** [comment]
haha, it is almost tomorrow(0:00) in my country.
Just do our best :)

**Comment by bil-elmoussaoui** [comment]
Sorry for not being able to do this, i think that adding the same effect for languages in the right side will be nice for packages as we won't need a page for only 4 distros (Arch, Fedora, Gnome & elementary) What do you think? @wi1d5ky

**Comment by wi1d5ky** [comment]
I agree with you that we won't need a page for only 4 distros, but I don't think adding the same effect would be good idea.

Here is what I thougnt:
When user clicks the `package` button, all things below would move down, and the four package would display between two buttons and features. If user click again, all things would change back.

**Comment by wi1d5ky** [comment]
@bil-elmoussaoui Maybe I could take this first, and you could beautify after my modification?
If you have time, maybe you could translate website first I think ;)

**Comment by bil-elmoussaoui** [comment]
I have some free time this week-end ! I think i can do both :)

**Comment by wi1d5ky** [comment]
Okay :D

**Comment by wi1d5ky** [comment]
Ping @bil-elmoussaoui 
Don't forget lollypop~~~

**Comment by bil-elmoussaoui** [comment]
Sorry i was a little busy, i will do that right now :P

**Comment by wi1d5ky** [comment]
Haha okay & Good luck :D

Bilal Elmoussaoui <notifications@github.com> 於 2015年11月23日 週一 下午6:48寫道：

**Comment by bil-elmoussaoui** [comment]
I used javascript, but the effect is really so bad. I used jQuery we might use it later for other effects #6
But the packages style looks really ugly. 
![capture du 2015-11-23 13-31-04](https://cloud.githubusercontent.com/assets/7660997/11336545/77cc7f8c-91e6-11e5-9671-d9272c8d6c46.png)

**Comment by bil-elmoussaoui** [comment]
Should i create a PR already or wait until you make the style of this looks better? what i propose is to use only the logo of each distro

**Comment by wi1d5ky** [comment]
Or maybe we could follow this site?
http://www.f2e.tw/contribute.html
Just see the Linux section, we could should a color for each distro.
How do you feel?

JQuery is okay. But if I could make it simpler, I will modify it latter :)

**Comment by bil-elmoussaoui** [comment]
The uncompressed version of jQuery is not that big (size). I don't think that having a lot of colours is great idea..

**Comment by wi1d5ky** [comment]
Don't worry, you could use jQuery first.

Well, I think you could make it more pretty, so just follow your mind & heart ;)

**Comment by bil-elmoussaoui** [comment]
Witth & without border radius 
![capture du 2015-11-23 14-12-28](https://cloud.githubusercontent.com/assets/7660997/11337322/7472a2ca-91ec-11e5-92da-520831d4b430.png)
![capture du 2015-11-23 14-13-58](https://cloud.githubusercontent.com/assets/7660997/11337323/74900342-91ec-11e5-9820-a1b39e4c89d7.png)
The colors are not hardcoded, you can modify that in the readme file :p do you have any idea for colours?

**Comment by wi1d5ky** [comment]
I think that we deserve better color :S

Or maybe we could give them these setting:

``` css
background: #fff;
border: 3px solid [some colors];
border-radius: 1.5rem;
```

If you think uniformity is important, you could use 2px, which is the border-width of download buttons.

**Comment by bil-elmoussaoui** [comment]
Yeah, the  colors were just to try :)

**Comment by wi1d5ky** [comment]
FYI, http://nipponcolors.com
Good luck!

**Comment by wi1d5ky** [comment]
ping @bil-elmoussaoui 
how is going on?

**Comment by bil-elmoussaoui** [comment]
I've been busy with studies..and i will pass my finals in two weeks.. i will be free in 19/January and i will fix all those issues :+1:

**Comment by wi1d5ky** [comment]
Good!

Bilal Elmoussaoui <notifications@github.com> 於 2015年12月20日 週日 下午11:02寫道：

**Comment by bil-elmoussaoui** [comment]
@wi1d5ky can you ping on me on telegram `@bilelmoussaoui`

**Comment by wi1d5ky** [comment]
Of course, I have ping you there.

**Comment by bil-elmoussaoui** [comment]
Fixed in my latest 4227000275a6bac230c909cbcb948dbbe8452047
